src/main/kotlin/com/bugsnag/android/gradle/BugsnagReleasesTask.kt

```diff
@@ -268,7 +269,8 @@ open class BugsnagReleasesTask @Inject constructor(
                 execSpec.standardOutput = baos
                 logging.captureStandardError(LogLevel.INFO)
             }
-            String(baos.toByteArray(), Charset.forName(CHARSET_UTF8)).trim { it <= ' ' }
+
+            baos.toString(Charset.defaultCharset()).trim { it <= ' ' }
```

Comment:
`ByteArrayOutputStream.toString(Charset)` is a Java 10 function that directly converts the internal byte array to a string instead of requiring a duplicate `byte[]`. AGP 8.+ require a minimum of Java 17